cmd/run.go

```diff
@@ -0,0 +1,90 @@
+package cmd
```

Comment:
Let's make it on par with devctl: https://github.com/giantswarm/devctl/tree/master/cmd

Reply:
Seems a bit of overkill to me for this purpose here.

Reply:
When you want to be the only developer for this piece of software I think you can do more or less what you want since you are responsible for it. When we want to collectively maintain software we should align with the standards we have consensus for.